src/auditwheel/policy/__init__.py

```diff
@@ -84,18 +84,18 @@ def architecture(self) -> Architecture:
         return self._architecture
 
     @property
-    def policies(self):
+    def policies(self):  # type: ignore[no-untyped-def]
         return self._policies
 
     @property
-    def priority_highest(self):
-        return max(p["priority"] for p in self._policies)
+    def priority_highest(self) -> int:
+        return max(int(p["priority"]) for p in self._policies)
```

Comment:
are this changes fully test covered? or the integration tests are enough for verifying them?

Reply:
both unit tests and integration tests are covering those: https://app.codecov.io/gh/pypa/auditwheel/pull/540/blob/src/auditwheel/policy/__init__.py

Reply:
One of the next todo will be to get some proper types for policies instead of dicts of Any which are a pain to work with in the first place.

Reply:
do we have any open issue for the TODO? I am accepting this PR